dbt/include/snowflake/macros/adapters.sql

```diff
@@ -155,14 +155,41 @@
     show terse objects in {{ schema_relation }}
   {%- endset -%}
 
+  {%- set fallback_sql -%}
+     select
+      created as "created_on",
+      table_name as "name",
+      case when table_type = 'BASE TABLE' then 'table'
+           when table_type = 'VIEW' then 'view'
+           when table_type = 'MATERIALIZED VIEW' then 'materializedview'
```

Comment:
Should this be `materializedview` or `materialized_view`? I think we use both for different reasons (I don't know why and wish we didn't), so we should make sure it's the right one.

Reply:
Hey @mikealfare thanks for reviewing this. I learned late yesterday that in fact we can paginate the `show terse objects` query on snowflake, so shifted direction there in working with @dbeatty10.